aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -116,6 +116,7 @@ pub enum FeatureFlag {
     DispatchableFungibleAsset,
     NewAccountsDefaultToFaAptStore,
     OperationsDefaultToFaAptStore,
+    MultisigV2Fix,
```

Comment:
nit: maybe a more descriptive name rather than "fix"? Would be nice to have some context for the future

Reply:
Done renaming it. Thanks!